sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -484,22 +484,29 @@ private TelemetryItemExporter createTelemetryItemExporter(StatsbeatModule statsb
                 LOGGER,
                 "Telemetry will not be stored to disk and retried on sporadic network failures");
 
-        TelemetryItemExporter telemetryItemExporter;
-        if (tempDir != null) {
-            telemetryItemExporter =
-                new TelemetryItemExporter(
-                    telemetryPipeline,
+        TelemetryPipelineListener telemetryPipelineListener;
+        if (tempDir == null) {
+            telemetryPipelineListener =
+                new DiagnosticTelemetryPipelineListener(
+                    "Sending telemetry to the ingestion service", false, " (telemetry will be lost)");
+        } else {
+            telemetryPipelineListener =
+                TelemetryPipelineListener.composite(
+                    // suppress warnings on retryable failures, in order to reduce sporadic/annoying
+                    // warnings when storing to disk and retrying shortly afterwards anyways
+                    // will log if that retry from disk fails
+                    new DiagnosticTelemetryPipelineListener(
+                        "Sending telemetry to the ingestion service",
+                        true,
+                        " (telemetry will be stored to disk and retried)"),
                     new LocalStorageTelemetryPipelineListener(
                         50, // default to 50MB
                         TempDirs.getSubDir(tempDir, "telemetry"),
                         telemetryPipeline,
-                        LocalStorageStats.noop(),
+                        statsbeatModule.getNonessentialStatsbeat(),
```

Comment:
we don't collect non-essential statsbeat in the standalone exporter. getNonessentialStatsbeat() will return null.

Reply:
ah, thx, I'm still not clear why we don't collect the same in both places, but will revert this change for this PR and we can revisit later